CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,134 @@
+# Changelog
+
+## [Unreleased](https://github.com/diem/diem/tree/HEAD)
+
+[Full Changelog](https://github.com/diem/diem/compare/diem-core-v1.3.0...HEAD)
+
+**Notable changes:**
+
+**[Consensus]**
+
+- Added decoupled execution. See related PRs ([\#8616](https://github.com/diem/diem/pull/8616), [\#8626](https://github.com/diem/diem/pull/8626), [\#8743](https://github.com/diem/diem/pull/8743), [\#8754](https://github.com/diem/diem/pull/8754), [\#8774](https://github.com/diem/diem/pull/8774), [\#8784](https://github.com/diem/diem/pull/8784), [\#8701](https://github.com/diem/diem/pull/8701), [\#8806](https://github.com/diem/diem/pull/8806), [\#8792](https://github.com/diem/diem/pull/8792), [\#8653](https://github.com/diem/diem/pull/8653))
+- \[decoupled-execution\] Add a new API for the TSafetyRules trait [\#8657](https://github.com/diem/diem/pull/8657)
+
+**[Diem Framework]**
+
+- Rename DiemID move resources and rust structures to VASPDomain [\#8695](https://github.com/diem/diem/pull/8695)
```

Comment:
The format of this document doesn't leave much room for explanations, but a comment would be in order here to explain that we chose to skip releasing version 1.3 of the Diem Framework. The DiemID resources were newly added in 1.3, so the renaming to VASPDomain can be done without breaking compatibility (at least on mainnet).

Reply:
I would capitalize "Move" (and also "Rust") but otherwise this looks good.

Reply:
Looks good, thanks!